### DISCUSSION TITLE
Add Open Url to Navigation

### DISCUSSION THREAD

**Discussion by nickrandolph** [unoplatform/uno.extensions#359] [Ideas]
Open Url should open the specified url in either the external browser, or a webview within the context of the application.

This could be used to display content such as a PDF without having to rely on third party libraries

**Comment by rkeithhill-keysight** [comment]
I found that this API also works on Linux. Have not tried macOS yet.

```csharp
Windows.System.Launcher.LaunchUriAsync(new Uri("http://www.microsoft.com"));
```
